creator-node/test/pollingTracks.test.js

```diff
@@ -1071,8 +1073,7 @@ describe('test Polling Tracks with real files', function () {
       .set('User-Id', userId)
       .send(trackMetadata)
       .expect(200)
-    trackMetadataMultihash = trackMetadataResp.body.data.metadataMultihash
-    trackMetadataFileUUID = trackMetadataResp.body.data.metadataFileUUID
+    const trackMetadataFileUUID = trackMetadataResp.body.data.metadataFileUUID
```

Comment:
these were never actually defined and mocha was now throwing an error with ts-node